tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -6769,9 +6769,9 @@ class ImplementingClass
         /* Comment */ otherTrait /* Comment */;
     /* Comment */ use /* Comment */ testTrait /* Comment */,
         /* Comment */ implementingTrait /* Comment */ /* Comment */ {
-        /* Comment */ A /* Comment */ /* Comment */ /* Comment */::testFunction insteadof /* Comment */ C; /* Comment */ /* Comment */
-        /* Comment */ B /* Comment */ /* Comment */ /* Comment */ /* Comment */::someOtherTestFunction as aliasedFunctionName; /* Comment */ /* Comment */
-        /* Comment */ C /* Comment */ /* Comment */ /* Comment */ /* Comment */ /* Comment */::testFunction as private testVisibility; /* Comment */
+        /* Comment */ A /* Comment */::/* Comment */ testFunction /* Comment */ insteadof /* Comment */ C; /* Comment */ /* Comment */
+        /* Comment */ B /* Comment */::/* Comment */ someOtherTestFunction /* Comment */ as /* Comment */ aliasedFunctionName; /* Comment */ /* Comment */
+        /* Comment */ C /* Comment */::/* Comment */ testFunction /* Comment */ /* Comment */ as private /* Comment */ testVisibility; /* Comment */
```

Comment:
Almost all comments in right position, except comments after `;`, but prettier use same logic and i think it is not bug here https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA3AhgJwATwM4zYC82A5PjKdgPQBUucB2t1N9Fz1A3NiADQgIABxgBLaHmSgsmCAHcAClgSSU6VBFEATfiABGmdGADWcGAGUhR0VADmyGJgCucAQAsYAWwA2AdTei+FZgcOYqgaKogQCeyOB4kgI2eHCYMAqGtp7oyABm6N4pAgBWeAAeAEKGJmbm6J5wADI2cHkFRSClZeY2tt5wAIpOEPBtha4gVpgpmHF66HrR3tC6Qpg2ML7aMG7IABwADAJrECm+hkJxa4ypqK0CAI7D8BnCqiDoeAC0UHBwWv9dJg4E9RMCMugsjkkPlxgIUp5RA5nBM8L1+kMRq0Ye0JjAFlstDtkAAmASOdCiby9ADCEE82TiUGg9xAThSABUFqpYSkAL58oA